src/main/resources/liquibase/crop_changelog/19_5_0.xml

```diff
@@ -20,4 +20,39 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="lkovacic" id="v19.5.0-2">
+		<preConditions onFail="MARK_RAN">
+			<and>
+				<sqlCheck expectedResult="0">select count(*) from cvterm where name = 'ENTRY_SOURCE';</sqlCheck>
```

Comment:
We should check by cv_id=1040  and is_obsolete = 0 given that's the unique key of the table 
@lik-83

Reply:
done